dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/publish/servers/servlet61_HttpResponseCodesTest/server.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for testing new optional HTTP response status code 308, 421, 422, 426 ">
```

Comment:
What do you mean by "optional"?

Reply:
they are not optional but new constant codes.  Updated.